src/sites/mod.rs

```diff
@@ -95,15 +97,23 @@ pub fn directory_keys_values(
 
                     validate_key_size(&key)?;
 
+                    // asset manifest should always contain all files
+                    asset_manifest.insert(url_safe_path, key.clone());
```

Comment:
What happens if there are duplicate keys? Will we upload both and the second gets overridden?

Reply:
can there be duplicates? these are based on files on disk no?

Reply:
I'm not familiar with this service :sweat_smile: if there can't be duplicates it's not worth worrying about.

Reply:
i'm not that familiar with it either, but the code suggests that it's iterating over files and creating a bundle to upload. the filepaths which ultimately determine the key & value would be unique